### DISCUSSION TITLE
Wall thickness problem

### DISCUSSION THREAD

**Discussion by tome9111991** [supermerill/SuperSlicer#1651] [General]
Hello,
i think is not really a slicer issue.
I have the problem with many prints that the walls are visibly thicker if there is no top infill or something similar.

![InkedIMG-20211016-WA0002_LI](https://user-images.githubusercontent.com/57866234/137590781-b8ead804-cd5d-4ab0-800a-054938fadcce.jpg)

the problem start at layer 264.
File
[red - Hat.zip](https://github.com/supermerill/SuperSlicer/files/7357681/red.-.Hat.zip)

this is an example from my last print.
have similar problems with other models.

is a problem with volumetric flow rate?

maybe someone has an idea?

**Comment by Botasky556** [comment]
This might be the same problem as "benchy hull line", if that's the problem, in short, there's no fix.

**Reply by tome9111991** [reply]
Ok thanks.
I have on benchy similar same

**Reply by supermerill** [reply]
From my tests, it comes from a difference in layer printing time. Look at the preview to see if it's aligned with the defect.

**Reply by tome9111991** [reply]
True.
In this case it cloud be layer printing time.
May have to tweak the settings.
